tests/math_builtin_api/CMakeLists.txt

```diff
@@ -18,33 +18,44 @@ set(math_builtin_depends
   "modules/test_generator.py"
 )
 
+set(SYCL_CTS_MATH_BUILTIN_N_SPLITS 16 CACHE STRING 
+    "The number of times to divide each math_builtin_api test")
+
 foreach(cat ${MATH_CAT_WITH_VARIANT})
   foreach(var ${MATH_VARIANT})
-    if ("${cat}" STREQUAL geometric AND "${var}" STREQUAL half)
-      continue()
-    endif()
+    foreach(split_index RANGE 1 ${SYCL_CTS_MATH_BUILTIN_N_SPLITS})
+      if ("${cat}" STREQUAL geometric AND "${var}" STREQUAL half)
```

Comment:
I guess that this `if` means that `geometric_half` is small enough by itself to be split, but a comment would be helpful here  about this exception